README.md

```diff
@@ -500,6 +500,7 @@ Some of the configuration values for the attributes above mentioned can be overr
 | TARGET_HOST          | config.resource.original.host       |
```

Comment:
CNR entry missing?

Reply:
fixed in https://github.com/telefonicaid/fiware-pep-steelskin/pull/419/commits/3a258f0ec7b2ab4a429e2439bf9d597fbd6d2403